app/components/CV/leftSideSections.tsx

```diff
@@ -0,0 +1,35 @@
+import { Section } from './section.js'
+
+type Props = {
+  contact: string[]
+  findMe: string[]
+  skills: string[]
+  technologies: string[]
+  courses: string[]
+  disclaimer: {
+    companyName: string
+    referenceNumber: string
+  }
+}
+
+const LeftSideSections = ({
+  contact,
+  findMe,
+  skills,
+  technologies,
+  courses,
+  disclaimer,
+}: Props) => {
+  return (
+    <>
+      <Section data={contact} name="contact" />
+      <Section data={findMe} name="find me" />
+      <Section data={skills} name="personal skills" />
+      <Section data={technologies} name="technologies" />
+      <Section data={courses} name="workshops & courses" />
```

Comment:
**issue**: Ensure that the `Section` component can handle cases where `disclaimer` is not provided to avoid potential runtime errors.